metadata_test.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestGithubRegexp(t *testing.T) {
```

Comment:
Can you mark this test as safe for parallel execution using `t.Parallel()` please?